ansible/roles/software/ckan/inventory/tasks/install.yml

```diff
@@ -56,14 +59,15 @@
     - etc/ckan/who.ini
     - etc/ckan/apache.wsgi
     - etc/ckan/datapusher.wsgi
-    - etc/apache2/sites-enabled/ckan.conf
-    - etc/apache2/sites-enabled/datapusher.conf
 
 - name: Configure production.ini
-  template: src=templates/{{ item }} dest=/{{ item }} owner=root group=www-data mode=0640
+  template: src={{ item }} dest=/{{ item }} owner=root group=www-data mode=0640
```

Comment:
Can you confirm that the `templates` wasn't working before because the `src=` was wrong?

Reply:
Ansible does some path magic ([checking different locations for files](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html?highlight=templates#using-roles)), so this was working correctly before. `templates` is the default (or the first location to check) for the `template` module, so this is essentially removing redundant path information, and leveraging the well-known default role templates directory.